### DISCUSSION TITLE
Continue Watching

### DISCUSSION THREAD

**Discussion by Zambucco** [damontecres/StashAppAndroidTV#444] [Ideas]
I have moved from Emby as StashAppAndroidTV has far more organisational capabilities.

I miss having a continue-watching section, I think it would be great if this were included in the opening page. Along with the 'Recently' sections.

Keep up the amazing work!

**Comment by damontecres** [comment]
Hey thanks!

If you have play history enabled (it is by default) and the minimum play percentage is low, you can get a good approximation of "continue watching" by creating a saved filter with `Resume Time` greater than `0:00` sorted by `Last Played At`, descending.

Then add that saved filter to the home page on the server and StashAppAndroidTV will also then display it on its main page.

**Reply by Zambucco** [reply]
I'm sorry for the really late reply. I had figured out how to use a resume time filter, but your small tweaks made a big difference.

Cheers

**Comment by Zambucco** [comment]
Further to the above, what are the most effective filter configurations for
'Watched'
'Unwatched'

**Reply by damontecres** [reply]
For `Unwatched`, I use: 
- Play Count is 0
- Play Duration is 0:00
- Resume Time is 0:00

I think `Watched` is more complicated because the web UI uses logical ANDs, so maybe:
- Play Duration is not 0:00
- Play Count is not 0
Because "Resume Time" could be zero after watching fully.

**Comment by Zambucco** [comment]
I am back! 

I am still struggling with this. Is there any chance that this could be made a feature? I have started using EMBY with the Stash plugin and it works wonderfully. But EMBY lacks so many of the features in Stash.

Stashapp has a plugin that puts a green tick in the scene card for watched, this doesn't work in your app though.

What I would really like is the ability to hide the played scenes.

**Reply by damontecres** [reply]
Sorry, what are you struggling with exactly? Are you having trouble setting the filters in-app? It might be easier to do it on your computer first time, save the filter, and use it in StashAppAndroidTV.
